src/main/java/me/botsko/prism/commands/DebugCommand.java

```diff
@@ -30,8 +30,21 @@
 
     @Override
     public void handle(CallInfo call) {
+        if(call.getArgs().length == 1) {
```

Comment:
Codacy found an issue: [WhitespaceAround: 'if' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)